bridger/menus/menus.py

```diff
@@ -55,7 +54,8 @@ def __iter__(self):
             yield "endpoint", endpoint
             if self.add:
                 self.add.request = request
-                yield "add", dict(self.add)
+                if dict(self.add):
+                    yield "add", dict(self.add)
```

Comment:
It's fine - but can you check in the tests? I think it is not covered any more.

Reply:
The tests succeed ( Although the test implying menu items have been commented out for some time now)